new-feature.txt

```diff
@@ -0,0 +1,3 @@
+New feature added in feature1 branch.
```

Comment:
Noob do you think this feature gonna work funcking change it asap.

Reply:
Hey its me. Why do you care ? Do your fucking course.